graphdatascience/tests/integration/test_graph_object.py

```diff
@@ -137,3 +137,11 @@ def test_graph_creation_time(G: Graph) -> None:
 
 def test_graph_modification_time(G: Graph) -> None:
     assert G.modification_time().year > 2000
+
+
+def test_graph_str(G: Graph) -> None:
+    assert str(G) == "Graph({'name': 'g', 'node_count': 4, 'relationship_count': 4})"
```

Comment:
This looks good, but I think the convention is to stylize like this (no big deal though):
```suggestion
    assert str(G) == "Graph(name='g', node_count=4, relationship_count=4)"
```